### ISSUE TITLE
Accessibility: lack of programmatic indication of expand/collapse and selected

### ISSUE THREAD

**Issue by 23jaredbarrett** [ProjectMirador/mirador#3578]
Throughout the  application, there is no programmatic indication of expand/collapse to screen readers.

**Places where this occurs:**

- Toggle sidebar
- language list dropdown (within "Workspace settings" submenu)
- Select workspace type lacks "aria-selected" attribute
- "Window Views & Thumbnail display" (right next to fullscreen button) lack of aria-selected attribute. Additionally, inactive interface components that are just titles "View" and "Thumbnails" are able to be keyboard focused.
- Sidebar: Index/Annotations lacks aria-selected state.
- Thumbnail panel at the bottom lack aria-selected.

**Recommendation:** Add aria-expanded to buttons that open submenus, and add aria-selected to elements where state is shown.

Side note: add label accessibility

**Comment by mferrarini** [comment]
Linking a similar issue reported by @brittnylapierre: https://github.com/ProjectMirador/mirador/issues/3883

**Comment by mferrarini** [comment]
@lutzhelm I can try to get to this today or by early next week!

**Comment by marlo-longley** [comment]
Developer action: take over the linked PR https://github.com/ProjectMirador/mirador/pull/3909, rebase it, and click to test that it closes this ticker
